src/gtimelog/tests/core/test_timelog.py

```diff
@@ -0,0 +1,319 @@
+import doctest
```

Comment:
I like my test file hierarchy to exactly mirror my code file hierarchy.

Therefore a TimeWindow class (defined in gtimelog.core.time) ought to be tested in gtimelog.tests.core.test_time, not gtimelog.tests.core.test_timelog.

Reply:
Fixed